README.md

```diff
@@ -290,6 +290,46 @@ There are many places in the format for compatible extensions:
 - Encodings: Encodings are specified by enum and more can be added in the future.
 - Page types: Additional page types can be added and safely skipped.
 
+### Thrift extensions
```

Comment:
We should perhaps move this to dedicated MD file, such that the examples can also be put in the same file?

Reply:
In the sync it was agreed that we put this small part here and pull only the examples in another file. I will wait for confirmation we want to move everything in another file before moving this. @julienledem WDYT?

Reply:
This does not look that small in the context of a README :-) And it seems a bit weird to me to feature it prominently in the README even though it's meant to be a rather obscure feature.

Reply:
How about we leave in the README just this:

> Parquet Thrift IDL reserves field-id `32767` of every Thrift struct for extensions. The (Thrift) type of this field is always `binary`.

Reply:
Yes, and link to the relevant document.

Reply:
Done.